tests/rkt_fetch_test.go

```diff
@@ -147,10 +147,6 @@ func TestImplicitFetch(t *testing.T) {
 			t.Fatalf("Cannot exec rkt: %v", err)
 		}
 
-		if err := child.Expect(foundMsg); err != nil {
-			t.Fatalf("Expected %q but not found: %v", foundMsg, err)
-		}
-
```

Comment:
Thinking more about it, the whole check `// 2. Try run/prepare with specified tag, should get the $foundMsg.` should be ignored/removed as it's scope is not reached (getting the $foundMsg).

Perhaps adding an option to the fetcher  and to rkt run/prepare/fetch (something like --no-cache) to disable the caching checks (do not check MaxAge and do not send ETag but continue saving them if received by the server) should be added? (in future, it should be useful also for the ux to force download without the need to firstly remove the image from the store...)